### ISSUE TITLE
TB1TFL anat scans for site MSSM do not exist

### ISSUE THREAD

**Issue by evaalonsoortiz** [spinal-cord-7t/coil-qc-code#43]
The data_processing.ipynb script currently segments the spinal cord on {subject}_acq-anat_TB1TFL.nii.gz scans. However, the site "MSSM" does not have those scans. TFL scans at that site only output "sub-MSSMxxx_acq-famp_TB1TFL", ie, the flip angle map. 

Possible solution: have an if case for that site and try to segment the spinal cord on the flip angle map

**Comment by jcohenadad** [comment]
Good catch. Indeed, we need an "if case". I'm not sure segmenting the cord on the FA map will work though. I need to look at various options.

**Comment by evaalonsoortiz** [comment]
My solution for now is the following:

I co-registered the UNIT1 SC segmentation to the TFL famp images using:
`
sct_apply_transfo -i ../anat/sub-MSSM3_UNIT1_seg_crop.nii.gz -d sub-MSSM3_acq-famp_TB1TFL.nii.gz -w warp_sub-MSSM3_inv-1_part-mag_MP2RAGE_crop2sub-MSSM3_acq-famp_TB1TFL.nii.gz -x linear -o sub-MSSM3_acq-famp_TB1TFL_seg.nii.gz`

I verified that the segmentation looked reasonable, then copied it to the manual labels folder:

`cp sub-MSSM3_acq-famp_TB1TFL_seg.nii.gz ../../derivatives/labels/sub-MSSM3/sub-MSSM3_acq-anat_TB1TFL_label-SC_seg.nii.gz
`

This way, few modifications are needed for the code.

**Comment by jcohenadad** [comment]
I agree this is a good solution. Once the OpenNeuro draft is published we can close this issue.

**Comment by Kyota-exe** [comment]
I created a new snapshot (v2.4.0) of the dataset for #48 after having uploaded the missing MP2RAGE scans for sub-MPI2 recently without knowing that the manual labels for this issue had been uploaded but that a new snapshot had not yet been created. As a result, v2.4.0 contains both the changes for #48 and for this issue, but I only mentioned the changes for #48 in the CHANGES file.

To fix this, I created a new version v2.4.1 with the corrected CHANGES file:
![image](https://github.com/spinal-cord-7t/coil-qc-code/assets/56048818/ee8b57a0-e83c-4c12-94b6-c81fccae927b)

Now that the manual labels for this issue have been uploaded and a snapshot has been created (v2.4.0/2.4.1), this issue should now be resolved.
